notebooks/cugraph_benchmarks/release.ipynb

```diff
@@ -0,0 +1,628 @@
+    "This notebook, starting with release 0.15, run all the various algorithms to compuite the performance gain.  \n",
+    "\n",
+    "### Algorithms\n",
+    "|        Algorithm        |  Graph   |   DiGraph   |\n",
+    "| ------------------------| -------- | ----------- |\n",
+    "| BFS                     |    X     |             |\n",
+    "| SSSP                    |    X     |             |\n",
+    "| PageRank                |          |      X      |\n",
+    "| WCC                     |          |      X      |\n",
+    "| Betweenness Centrality  |    X     |             |\n",
+    "| Louvain                 |    X     |             |\n",
+    "| Triangle Counting       |    X     |             |\n",
+    "\n",
+    "### Test Data\n",
+    "\n",
+    "| File Name              | Num of Vertices | Num of Edges |\n",
+    "| ---------------------- | --------------: | -----------: |\n",
+    "| preferentialAttachment |         100,000 |      999,970 |\n",
+    "| dblp-2010              |         326,186 |    1,615,400 |\n",
+    "| coPapersCiteseer       |         434,102 |   32,073,440 |\n",
+    "| as-Skitter             |       1,696,415 |   22,190,596 |\n",
+    "\n",
+    "\n",
+    "Notebook Credits\n",
+    "\n",
+    "    Original Authors: Bradley Rees\n",
+    "    Last Edit: 08/17/2020\n",
+    "    \n",
+    "RAPIDS Versions: 0.15\n",
+    "\n",
+    "Test Hardware\n",
+    "    GV100 32G, CUDA 10.2\n",
+    "    Intel(R) Core(TM) CPU i7-7800X @ 3.50GHz\n",
+    "    32GB system memory\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Timing \n",
+    "What is not timed:  Reading the data</p>\n",
+    "What is timmed: (1) creating a Graph, (2) running the algorithm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# system and other\n",
+    "import gc\n",
+    "import os\n",
+    "import time\n",
+    "import numpy as np\n",
+    "\n",
+    "# rapids\n",
+    "import cugraph\n",
+    "import cudf\n",
+    "\n",
+    "# NetworkX libraries\n",
+    "import networkx as nx\n",
+    "\n",
+    "# MTX file reader\n",
+    "from scipy.io import mmread"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try: \n",
+    "    import community\n",
+    "except ModuleNotFoundError:\n",
+    "    os.system('pip install python-louvain')\n",
+    "    import community"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try: \n",
+    "    import matplotlib\n",
+    "except ModuleNotFoundError:\n",
+    "    os.system('pip install matplotlib')\n",
+    "\n",
+    "import matplotlib.pyplot as plt; plt.rcdefaults()\n"
```

Comment:
can't find where it is used

Reply:
I was going to generator a plot at the bottom like in the other notebooks.  That it a pending update